TLM/TLM/State/ConfigData/DebugSettings.cs

```diff
@@ -94,91 +54,49 @@ public class DebugSettings {
         /// This adds access to the new moved type from the compatible old field
         /// </summary>
         // Property will not be serialized, permit use of obsolete symbol
-#pragma warning disable 612
+        [Obsolete]
         public API.Traffic.Enums.ExtVehicleType ApiExtVehicleType
             => LegacyExtVehicleType.ToNew(ExtVehicleType);
-#pragma warning restore 612
 
         public ExtPathMode ExtPathMode = ExtPathMode.None;
     }
 
-    /// <summary>
-    /// Indexes into Debug.Switches
-    /// </summary>
+    [Flags]
```

Comment:
Why bit flags and not arraysof boolean? Is there any need to pack all bits into 1 integer?

Reply:
it makes more sense this way. why to write extra code? why to have code duplication?
and we are not serializing anything anymore so there is no need for array.

Reply:
Reason is that checking a processor-register wide boolean (64 bit) is faster than reading and masking a bit, with same result. Not important if all code for this is deleted in a Release build.

Reply:
Yes that code won't run in Release build, at least it shouldn't

Reply:
I am just trying to keep it simple.